boards/lilygo/ttgo_t7v1_5/doc/index.rst

```diff
@@ -0,0 +1,222 @@
+.. zephyr:board:: ttgo_t7v1_5
+
+Overview
+********
+
+LILYGO® TTGO T7 Mini32 V1.5 ia an IoT mini development board
+based on the Espressif ESP32-WROVER-E module.
+
+It features the following integrated components:
+- ESP32 chip (240MHz dual core, 520KB SRAM, Wi-Fi, Bluetooth)
+- on board antenna
+- Micro-USB connector for power and communication
+- JST GH 2-pin battery connector
+- LED
+
+Functional Description
+**********************
+This board is based on the ESP32-WROVER-E module with 4MB of flash (there
+are models 16MB as well), WiFi and BLE support. It has a Micro-USB port for
+programming and debugging, integrated battery charging and an on-board antenna.
+
+Connections and IOs
+===================
+
+The ``ttgo_t7v1_5`` board target supports the following hardware features:
+
++-----------+------------+------------------+
+| Interface | Controller | Driver/Component |
++===========+============+==================+
+| CPU       | ESP32      | arch/xtensa      |
++-----------+------------+------------------+
+| GPIO      | on-chip    | gpio_esp32       |
++-----------+------------+------------------+
+| UART      | on-chip    | uart_esp32       |
++-----------+------------+------------------+
+| I2C       | on-chip    | i2c_esp32        |
++-----------+------------+------------------+
+| SPI       | on-chip    | spi_esp32_spim   |
++-----------+------------+------------------+
+| LoRa      | SX1276     | lora_sx127x      |
++-----------+------------+------------------+
+| WiFi      | on-chip    | wifi_esp32       |
++-----------+------------+------------------+
+| BLE       | on-chip    | bluetooth_esp32  |
++-----------+------------+------------------+
+| Flash     | on-chip    | flash_esp32      |
++-----------+------------+------------------+
+
+Start Application Development
+*****************************
+
+Before powering up your Lilygo ``ttgo_t7v1_5``, please make sure that the board is in good
+condition with no obvious signs of damage.
```

Comment:
is this really needed?

Reply:
no, took it from docs from another lilygo board. removed.